spec/features/schools/participants/update_participants_details_spec.rb

```diff
@@ -7,6 +7,7 @@
   include ManageTrainingSteps
 
   before do
+    travel_to Date.new(2022, 9, 1)
```

Comment:
Could use the `travel_to: Date.new(2022, 9, 1)` on the describe or scenario blocks so that it never leaks?